_opensearch/query-dsl/geo-bounding-box.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: default
+title: Geo-bounding box queries
+parent: Query DSL
+nav_order: 55
+---
+
+# Geo-bounding box queries
+
+To search for documents that contain [geopoint]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-point) fields, use a geo-bounding box query. The geo-bounding box query returns documents whose geopoints are within the bounding box specified in the query. A document with multiple geopoints matches the query if at least one geopoint is within the bounding box.
+
+## Example
+
+You can use a geo-bounding box query to search for documents that contain geopoints. 
+
+Create a mapping with the `point` field is mapped as `geo_point`:
```

Comment:
"Create a mapping where the `point` field is mapped as `geo_point`:"
Or,
"Create a mapping with the `point` field mapped as `geo_point`:"